models/BillingStructure.v1.json

```diff
@@ -1,12 +1,12 @@
 {
   "title": "BillingStructure",
   "type": "object",
-  "description": "Structure de facturation du partenaire.",
+  "description": "Structure de facturation.",
   "properties": {
-    "partner_billing": {
+    "billing_structure": {
```

Comment:
Why "structure"? Sound a bit weird to me.
it's like naming a class "class billinClass"